hips/hip-00XX.md

```diff
@@ -0,0 +1,213 @@
+---
+hip: 9999
+title: "Utilize Server Side Apply for installs/upgrades"
+authors: [ "George Jenkins <gvjenkins@gmail.com>" ]
+created: "2023-07-07"
+type: "feature"
+status: "draft"
+helm-version: "4"
+---
+
+## Abstract
+
+<!-- A short (~200 word) description of the technical issue being addressed. -->
+This HIP proposes for Helm to support Kubernetes' [Server-Side Apply](https://kubernetes.io/docs/reference/using-api/server-side-apply/) (SSA) for [object](https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-object) management.
+
+Helm 3 introduced the [three-way strategic merge and patch](https://helm.sh/docs/faq/changes_since_helm2/#improved-upgrade-strategy-3-way-strategic-merge-patches).
+Allowing Helm to update objects which have been modified by other processes.
+Kubernetes now offers a similar server-side process that has several advantages over the client-side apply (CSA) methods that Helm and `kubectl` (for example) have traditionally utilized.
+
+## Motivation
```

Comment:
I should also include that client-side validation is now largely unsupported (need to read up on the details):

https://github.com/kubernetes/kubectl/issues/1625